docs/components/concepts/expressions.md

```diff
@@ -180,7 +180,14 @@ More functions for string values are available as [built-in string functions](/c
 
 ### Temporal expressions
 
-The current date and date-time can be accessed using the built-in functions `today()` and `now()`. To store the current date or date-time in a variable, convert it to a string using the built-in function `string()`.
+The current date and date-time can be accessed using the built-in functions `today()` and `now()`.
+Storing a temporal value as a variable will store it in the ISO 8601 format.
+
+:::note
+The timezone in the ISO 8601 format differs from the FEEL specific format. Storing a date-time as a process variable means
+you can not use it in temporal expressions later on in the process. To circumvent this we recommend storing
+date-times in the FEEL specific format using the built-in `string()` function.
```

Comment:
❓ Is this true? I [thought the variables contain exactly the same string representation](https://github.com/camunda/zeebe/pull/8842/files#diff-73d88a5d343f1206fdecd30b6d473ca53a1c966d53a6c8934b23f5f842188f92R105-R139) that can be used to create the FEEL temporal values in the first place. Why can't they be used in later temporal expressions?

Reply:
I based this on this comment on the issue 🤷 
https://github.com/camunda/camunda-platform-docs/issues/650#issuecomment-1344075449

Reply:
I see. After reading that issue, it sounds like we should just store the temporal value in FEEL format, not in ISO 8601. That seems like the cleanest solution, IMO. 

Perhaps we should document it immediately in that way then 🤔 But then we document the behavior that we don't have because of a bug... Now I'm not sure about documenting this before fixing that bug 😆 WDYT?

Reply:
I think I want to be done with this issue 😄

From what I can see what I've documented now is correct. When we fix the bug we can fix the documentation as well 🤷‍♂️

I don't like documenting that something works a specific way if we know it doesn't. That's only going to cause confusion and issues.

Reply:
> When we fix the bug we can fix the documentation as well 🤷‍♂️

The problem with this is that we typically try not to change the behavior written in the docs. If we have bugs, then we should fix those, but the documented behavior should not change.

>I don't like documenting that something works a specific way if we know it doesn't. That's only going to cause confusion and issues.

Agreed. So, I feel that we should either not document it yet, or document it as we want it, with a clear not on which part doesn't work yet ([only using timezone offset seems to work right now](https://github.com/camunda/zeebe/issues/10307#issuecomment-1433373633))